Snowdog_Components/components/Molecules/table/_table.scss

```diff
@@ -98,14 +98,6 @@
 
                 td {
                     border-bottom: 0;
```

Comment:
what about this? I think it's useless because td doesn't have borders by default